launchpod/angular-launchpod/src/app/create-form/create-form.component.ts

```diff
@@ -16,7 +15,8 @@ export class CreateFormComponent implements OnInit {
 
   constructor(private formHandlerService: FormHandlerService) {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+  }
```

Comment:
Revert back to one line?

Reply:
sorry, generating the component from the cli makes it look like this lol. fixed it!